jax/api.py

```diff
@@ -2074,3 +2074,49 @@ def fun_invertible(*args, **kwargs):
                                    concrete=concrete)
     return tree_unflatten(out_tree(), out_flat)
   return fun_invertible
+
+def make_hlo(fun: Callable,
+             optimize: bool = False,
+             metadata: bool = False, 
+             platform: Optional[str] = None,
+             num_replicas: int = 1,
+             num_partitions: int = 1,
+             **xla_computation_kwargs) -> Callable:
+  """Utility function for printing JAX-emitted HLO and XLA-compiled HLO.
```

Comment:
Right now we also have `jax.xla_computation(fun)(args).as_hlo_text()`. I understand that the new function returns the same thing if `optimize = False`?